x-pack/plugins/apm/server/lib/search_strategies/correlations/async_search_service.ts

```diff
@@ -28,7 +30,9 @@ const currentTimeAsString = () => new Date().toISOString();
 
 export const asyncSearchServiceProvider = (
   esClient: ElasticsearchClient,
-  params: SearchServiceParams
+  getApmIndices: () => Promise<ApmIndicesConfig>,
+  searchServiceParams: SearchServiceParams,
+  includeFrozen: boolean
```

Comment:
Is `includeFrozen` being used here?

Reply:
Just re-read the post description and my question is redundant here :) Sorry the noise